charts/sn-platform/templates/toolset/_toolset.tpl

```diff
@@ -199,3 +199,59 @@ Define toolset TLS certificate secret name
 {{ .Release.Name }}-{{ .Values.tls.toolset.cert_name }}
 {{- end -}}
 {{- end -}}
+
+{{/*
+Define the toolset web service url
+*/}}
+{{- define "toolset.web.service.url" -}}
+{{- if not .Values.toolset.useProxy -}}
+{{- if and .Values.tls.enabled .Values.tls.broker.enabled -}}
+https://{{ template "pulsar.fullname" . }}-{{ .Values.broker.component }}:{{ .Values.broker.ports.https }}
+{{- else -}}
+http://{{ template "pulsar.fullname" . }}-{{ .Values.broker.component }}:{{ .Values.broker.ports.http }}
+{{- end -}}
+{{- else -}}
+{{- if and .Values.tls.enabled .Values.tls.proxy.enabled -}}
+https://{{ template "pulsar.fullname" . }}-{{ .Values.proxy.component }}:{{ .Values.proxy.ports.https }}
+{{- else -}}
+http://{{ template "pulsar.fullname" . }}-{{ .Values.proxy.component }}:{{ .Values.proxy.ports.http }}
+{{- end -}}
+{{- end -}}
+{{- end -}}
+
+{{/*
+Define the toolset broker service url
+*/}}
+{{- define "toolset.broker.service.url" -}}
```

Comment:
are we using this template anywhere? couldn't find it in this pr.

Reply:
Removed temporarily unused code,